ui/src/App/Main.jsx

```diff
@@ -45,6 +45,9 @@ import isLoggedInQuery from './isLoggedIn.graphql';
     ':not(pre) > code': {
       ...theme.mixins.highlight,
     },
+    '.mmJN4cVbficDwBMDQAPWE': {
+      position: 'absolute',
+    },
```

Comment:
Nice find! Seems to work but we should probably add this change in https://github.com/taskcluster/react-schema-viewer so that it works for everyone using that library, not just here. Does this make sense to you? Let me know if you had questions.

Reply:
Yeah that's true

Reply:
> To achieve that, should I instead fork that repository, add the fix and create a PR?

Sounds like a good idea  to me. I would fork then clone the repo.

Reply:
Here's a link to the PR
https://github.com/taskcluster/react-schema-viewer/pull/37